### ISSUE TITLE
imported and not used

### ISSUE THREAD

**Issue by FuzzOli87** [zchee/nvim-go#17]
Hello,

I'm getting an error when I'm running ```make``` on the plugin.
```
lint.go:19:2: imported and not used: "nvim-go/log"
nvim-go/command/delve
FATAL: command "build" failed: exit status 2
```

**Comment by akatrevorjay** [comment]
https://github.com/zchee/nvim-go/pull/18

**Comment by zchee** [comment]
@FuzzOli87 @akatrevorjay merged #18. Thanks!
